TAScheduler/UnitTests/tests.py

```diff
@@ -11,38 +13,165 @@
 # Feel free to make suggestions on discord (add/remove/edit methods)!.
 ### Rememeber: These methods were made before any coding (I was guessing) so it's likely they should be changed.
 class TestUserLogin(TestCase):  # Alec
-    pass
-
-
-class TestUserGetID(TestCase):  # Alec
-    pass
+    def setUp(self):
+        self.user = User.objects.create(
+            email_address="test@example.com",
+            password="password123",
+            first_name="Test",
+            last_name="User",
+            home_address="123 Street",
+            phone_number=1234567890
+        )
 
+    def test_login_valid_credentials(self):
+        result = self.user.login("test@example.com", "password123")
+        self.assertTrue(result)
```

Comment:
I think this would be assertRedirect because a failed login would go back to the login page but that could be a different test

Reply:
good point, fixed